velox/docs/develop/dynamic-loading.rst

```diff
@@ -12,19 +12,19 @@ Getting Started
 
 1. **Create a C++ file for your dynamic library**
 
-   For dynamically loaded function registration, the format followed mirrors that of built-in function registration with some noted differences. Using `DynamicTestFunction.cpp` as an example, the function uses the `extern "C"` keyword to protect against name mangling. A `registry()` function call is also necessary.
+   For dynamically loaded function registration, the format followed mirrors that of built-in function registration with some noted differences. Using `DynamicTestFunction.cpp` as an example, the function uses the `extern "C"` keyword to protect against name mangling. A `registry()` function call is also necessary. The `registry` function name is customizable.
```

Comment:
Please change this to make the main example use a non-default registry function name, and then the default name is a special case..